Resources/Locale/en-US/procedural/expeditions.ftl

```diff
@@ -44,5 +44,13 @@ salvage-expedition-not-all-present = Not all salvagers are aboard the shuttle!
 salvage-expedition-announcement-countdown-minutes = {$duration} minutes remaining to complete the expedition.
 salvage-expedition-announcement-countdown-seconds = {$duration} seconds remaining to complete the expedition.
 salvage-expedition-announcement-dungeon = Dungeon is located {$direction}.
+salvage-expedition-direction-west = west
+salvage-expedition-direction-south = south
+salvage-expedition-direction-north = north
+salvage-expedition-direction-east = east
+salvage-expedition-direction-southeast = south-east
+salvage-expedition-direction-northeast = north-east
+salvage-expedition-direction-northwest = north-west
+salvage-expedition-direction-southwest = south-west
```

Comment:
i think u might wanna move em to `generic.ftl` so you don't rewrite those strings every time

Reply:
I thought so too, but in Russian these sides should have different endings for these cases, if in the case of a shuttle we just say the side, then in the case of an expedition we need to say a little differently. As an option, both notifications can be reduced to one format, then this can be implemented.

Reply:
> I thought so too, but in Russian these sides should have different endings for these cases, if in the case of a shuttle we just say the side, then in the case of an expedition we need to say a little differently. As an option, both notifications can be reduced to one format, then this can be implemented.

This is the english translation. just pass the direction into the translation directly and use the `LOC` fluent function to specialize it for your translation, or in our case not specialize it.